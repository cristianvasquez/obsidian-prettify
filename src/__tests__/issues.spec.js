const prettifier = require('../prettifier')


// @TODO
// Read with base __DIRNAME
// run_spec(__dirname, ["markdown"], { proseWrap: "always" });

describe("issues", () => {


    test("https://github.com/cristianvasquez/obsidian-prettify/issues/1", () => {
        const content = `
# issues/1

Prettifier:
- [ ] Don't break my TODO
            `
        return prettifier(content).then(data => {
            expect(data).toMatchSnapshot();
        });
    });

    test("https://github.com/cristianvasquez/obsidian-prettify/issues/1", () => {
        const content = `
# issues/1

1. fjlkefjl
    1. feafe
2. feafea
3. feaf
    1. fea
3. fefe
            `
        return prettifier(content).then(data => {
            expect(data).toMatchSnapshot();
        });
    });

    test("https://github.com/cristianvasquez/obsidian-prettify/issues/12", () => {
        const content = `
Please honor backslashes \\
            `
        return prettifier(content).then(data => {
            expect(data).toMatchSnapshot();
        });
    });

});
