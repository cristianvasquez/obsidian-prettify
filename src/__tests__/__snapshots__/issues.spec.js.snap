// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`issues https://github.com/cristianvasquez/obsidian-prettify/issues/1 1`] = `
VFile {
  "contents": "# issues/1

Prettifier:

-   [ ] Don't break my TODO
",
  "cwd": "/home/cvasquez/obsidian/workspace/.obsidian/plugins/markdown-prettifier",
  "data": Object {},
  "history": Array [],
  "messages": Array [],
}
`;

exports[`issues https://github.com/cristianvasquez/obsidian-prettify/issues/1 2`] = `
VFile {
  "contents": "# issues/1

1.  fjlkefjl
    1.  feafe
2.  feafea
3.  feaf
    1.  fea
4.  fefe
",
  "cwd": "/home/cvasquez/obsidian/workspace/.obsidian/plugins/markdown-prettifier",
  "data": Object {},
  "history": Array [],
  "messages": Array [],
}
`;

exports[`issues https://github.com/cristianvasquez/obsidian-prettify/issues/12 1`] = `
VFile {
  "contents": "Please honor backslashes \\\\
",
  "cwd": "/home/cvasquez/obsidian/workspace/.obsidian/plugins/markdown-prettifier",
  "data": Object {},
  "history": Array [],
  "messages": Array [],
}
`;
